code/modules/clothing/head/_head.dm

```diff
@@ -7,6 +7,7 @@
 	slot_flags = ITEM_SLOT_HEAD
 	var/blockTracking = 0 //For AI tracking
 	var/can_toggle = null
+	var/snug_fit = FALSE //can the hat be knocked off?

```

Comment:
This is confusing, it leads people to think that true means it's able to be knocked off

Reply:
That's correct, `TRUE` = the hat can fall off

Reply:
That's what it makes us assume, but if you look at the code, all important helmets and hats have snug_fit = TRUE and the code has a conditional of (!snug_fit). So either change it to (snug_fit) and make all the important helmets to false or change the var name and/or the commentary

Reply:
snug (snŭg)►
adj.	Comfortably sheltered; cozy.
adj.	Small but well arranged: a snug apartment. See Synonyms at comfortable.
adj.	Closely secured and well built; compact: a snug little sailboat.

i don't see the confusion with the var name

Reply:
What about the "//can the hat be knocked off?"?

Reply:
the confusion is with the comment. when you read "can the hat be knocked off?" and then read "YES", you'll think "yes, the hat can be knocked off." the comment should read "is the hat immune to being knocked off?"